lib/seahorn/BvOpSem2MemRepr.cc

```diff
@@ -13,6 +13,12 @@ auto as_std_array(const T &t, const Rest &... rest) ->
 namespace seahorn {
 namespace details {
 
+// MemSet operates at word level.
+// _val must fit within a byte
+// _val is converted to a byte.
```

Comment:
I think this comment is more about implementation than use, so it should be inside the method